swarm/api/routes/boundary.py

```
@@ -399,6 +400,20 @@ async def get_boundary_review(
+    if flow_key:
+        try:
+            validate_path_component(flow_key, "flow_key")
+        except ValueError as e:
+            raise HTTPException(
+                status_code=400,
+                detail={
+                    "error": "invalid_flow_key",
+                    "message": str(e),
+                    "details": {"flow_key": flow_key},
+                },
+            )
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)